src/diffusers/quantizers/torchao/torchao_quantizer.py

```diff
@@ -33,23 +33,29 @@
 
 if is_torch_available():
     import torch
-    import torch.nn as nn
-
-    SUPPORTED_TORCH_DTYPES_FOR_QUANTIZATION = (
-        # At the moment, only int8 is supported for integer quantization dtypes.
-        # In Torch 2.6, int1-int7 will be introduced, so this can be visited in the future
-        # to support more quantization methods, such as intx_weight_only.
-        torch.int8,
-        torch.float8_e4m3fn,
-        torch.float8_e5m2,
-        torch.uint1,
-        torch.uint2,
-        torch.uint3,
-        torch.uint4,
-        torch.uint5,
-        torch.uint6,
-        torch.uint7,
-    )
+
+    if version.parse(torch.__version__) >= version.parse('2.6'):
```

Comment:
oh thanks so much!
can we use `is_torch_version`? https://github.com/huggingface/diffusers/blob/c1e7fd5b3423349cbfa13b136eb262a49d113ec3/src/diffusers/utils/import_utils.py#L727

Reply:
Thank you! Apologies that I missed this 😅 

I think this should be something like:

```
if version <= 2.5:
    # do not use any of the uintx
else:
    # use the uintx types as well
```

I believe the uintx data types were added in 2.5 and not 2.6

Reply:
Also maybe good to have another branch that adds `float8_e4m3fn` and `float8_e5m2` only if torch version is `>= 2.2`. `int8` came in `2.0` I believe but since minimum diffusers requirement is `torch>=1.4`, another branch for it might be needed. LMK if I can help with any changes